src/rev_ai/utils.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using Rev.ai"""
+
+from . import CustomVocabulary
+
+
+def _process_vocabularies(unprocessed_vocabularies):
+    return list(map(lambda custom_vocabulary: custom_vocabulary.to_dict()
+                    if isinstance(custom_vocabulary, CustomVocabulary)
+                    else custom_vocabulary, unprocessed_vocabularies))
```

Comment:
Should we have a check to make sure that the unprocessed vocab has the key "phrases"?

Reply:
Validation should be done by the REST API. It's more flexible for development. Validating by API reduces the amount of validation code we need to rewrite in the event of an API change.

@KostasRev please remove sdk (client) side validation.

Reply:
The API will return an error with the correct error message in the event the request is malformed.

Reply:
Alright, will do.